docker/Dockerfile-workers

```diff
@@ -34,10 +34,14 @@ RUN dnf -y install \
     python3-devel \
     python3-pip \
     skopeo \
+    net-tools \
     && dnf clean all
 
 COPY --from=builder /usr/bin/opm /usr/bin/opm
 ADD https://github.com/estesp/manifest-tool/releases/download/v1.0.0/manifest-tool-linux-amd64 /usr/bin/manifest-tool
+RUN curl -LJO https://github.com/fullstorydev/grpcurl/releases/download/v1.7.0/grpcurl_1.7.0_linux_x86_64.tar.gz && \
+    tar -zxvf grpcurl_1.7.0_linux_x86_64.tar.gz && \
+    cp grpcurl /usr/bin
```

Comment:
I think that, Ansible script has to be modified as well.

Reply:
I have created additional MRs